lib/blockscore/actions/retrieve.rb

```diff
@@ -12,7 +12,7 @@ module Actions
     module Retrieve
       module ClassMethods
         def retrieve(id, options = {})
-          fail ArgumentError if id.empty?
+          fail ArgumentError,'ID must be supplied' if id.nil? || id.empty?
```

Comment:
Ok - I agree. So what is the best procedure for me to do that now that I have an active PR:

- Switch to my branch
- Adjust the code and commit

Now do I need to squash the commits on my end?